app/src/main/java/org/dicekeys/app/data/DiceKeyDescription.kt

```diff
@@ -0,0 +1,18 @@
+package org.dicekeys.app.data
+
+import org.dicekeys.app.encryption.EncryptedDiceKey
+import org.dicekeys.dicekey.DiceKey
+import org.dicekeys.dicekey.Face
+
+data class DiceKeyDescription constructor(val keyId: String, val centerFace: String) {
+    constructor(diceKey: DiceKey<*>) : this(
+        diceKey.keyId,
+        diceKey.centerFace().toHumanReadableForm(true)
```

Comment:
Needs documentation as to the orientation of the center face and why the orientation is stored.

The TypeScript code keeps only the center die's letter and digit, presenting it always upright and therefore not needing the orientation.  If you're always keeping the orientation of the center die upright and it's just convenient to include the whole orientation in the data structure rather than write specialized code to remove it, document that.  If you're not rotating the DiceKey into a center-die-upright orientation before grabbing that center face, the orientation is relative to the other canonicalization method or just how the key was scanned, and you're then ignoring the orientation when displaying, please say that.

Reply:
This is a helper class to keep a describe DiceKey, combine EncryptedDiceKey and DiceKey only for display purposes. It is not stored.

The DiceKeyCenterFaceOnlyView takes a Face to display the center dice key.
https://github.com/dicekeys/dicekeys-android/blob/9106832af7a4dd6822c4576b81c996a6fec077fb/app/src/main/java/org/dicekeys/app/views/DiceKeyCenterFaceOnlyView.kt#L17

A face can only be constructed by providing a 3 char string.
https://github.com/dicekeys/dicekeys-android/blob/9106832af7a4dd6822c4576b81c996a6fec077fb/dicekey/src/main/java/org/dicekeys/dicekey/Face.kt#L60-L63

Also, is the same way we store it on the EncryptedStorage.
https://github.com/dicekeys/dicekeys-android/blob/9106832af7a4dd6822c4576b81c996a6fec077fb/app/src/main/java/org/dicekeys/app/encryption/EncryptedStorage.kt#L45-L50

Reply:
In the TypeScript app, we are now always displaying DiceKeys rotated so that the center face is upward.  For cryptographic operations we use the other canonicalization since that's what we standardized when people started using the product.  But for display, always center face upward.

So, for presenting an obscured DiceKey, with just the centered die, the orientation has no meaning, because we always present it as upright.  So, this data structure should either not contain orientation, or document that the orientation is useless and ignored but there for ease of implementation, or do something to explain what orientation means in this context.

Reply:
Add #159

Reply:
Just to clarify, it is stored with the orientation, but the UI always display the center die without orientation.

Reply:
So if the field is effectively unused, please document that in the code